docs/src/BHM-example.md

```diff
@@ -130,29 +129,50 @@ println(t_strat)
 Finally, we can start the main FCIQMC loop:
 
 ```@example BHM-example
-r_strat = ReportDFAndInfo(k = k, projector = UniformProjector(), writeinfo = false) # hide
 df = lomc!(Ĥ,svec;
             params = params,
             laststep = steps_equilibrate + steps_measure,
             s_strat = s_strat,
             r_strat = r_strat,
-            τ_strat = t_strat)
+            τ_strat = t_strat);
 println("Writing data to disk...")
 ```
 
-Saving output data stored in `df.df` into a `.feather` file which can be read in later:
+Saving output data stored in `df.df` into a `.arrow` file which can be read in later:
 
 ```@example BHM-example
-Feather.write("fciqmcdata.feather", df.df)
+Arrow.write("fciqmcdata.arrow", df.df)
```

Comment:
follow on change:
```suggestion
Arrow.write("fciqmcdata.arrow", df)
```
The same things will need to be changed in the actual script as well.

Just a note, instead of of importing `Arrow.jl` you could also do
```julia
save_df("fciqmcdata.arrow", df)
```
and later load it using `load_df()`. It will do the same thing, though.

Reply:
Changed to the suggested syntax. It does need the `using Rimu.RimuIO` to work, right?